README.md

```diff
@@ -0,0 +1,148 @@
+# Manual to add openedx_nodebb_discusson in your edx platform
+
+Clone this repo into `openedx_nodebb_discussion` folder using the given command
+
+`git clone https://github.com/arbisoft/openedx-nodebb-discussion.git openedx_nodebb_discussion`
+
+---
+
+Copy this folder and place this at the following location in edx codebase
+
+`../edx-platform/openedx/features/`
+
+---
+
+Add `ENABLE_NODEBB_DISCUSSION` flag in the `FEATURES` in the following files
+
+
+- lms.env.json
+- cms.env.json
+
+
+and set their values to   `true`
+
+```
+FEATURES = {
+    ...
+    "ENABLE_NODEBB_DISCUSSION": true,
+}
+```
+---
+
+Add the following dictionary at the end of `lms.env.json` and `cms.env.json`
+
+```
+{
+    ...
+    ...
+    "NODEBB_SETTINGS": {
+        "URL": "<YOUR NODEBB URL>",
+        "DOMAIN": "<YOUR NODEBB DOMAIN>",
+        "TASK_RETRY_DELAY": <int: Seconds>,
+    }
+}
+```
+
+> **Note:** `TASK_RETRY_DELAY` is seconds. NodeBB Client has to wait to make a call to API again in case of server error.
```

Comment:
Add description that this is a celery task and the retry delay is the celery delay.

Reply:
Done